best_line_test.cpp

```diff
@@ -0,0 +1,46 @@
+#include "best_line.h"
+
+#include <gtest/gtest.h>
+
+TEST(GetBestLine, RegularLine) {
+  vector<Point> pts = {Point(1, 3), Point(2, 1),   Point(3, -1),
```

Comment:
This is pretty good, although I would make it
const std::vector<Point> pts = {{1, 3}, {2, 1}, {3, -1}, ...}